packages/admin-stories/src/docs/comet/admin/errors/errordialog.stories.mdx

```diff
@@ -0,0 +1,128 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import { errorDialogStoryProviderDecorator } from "../../../../error-dialog-provider.decorator";
+import { apolloSwapiStoryDecorator } from "../../../../apollo-swapi-story.decorator";
+
+import { ErrorDialogContext } from "@comet/admin";
+import { Button, TextField } from "@material-ui/core";
+
+<Meta title="docs/@comet/admin/errors/errordialog" />
+
+# ErrorDialog
+
+## Setup
+
+Setup is easy, just add the `ErrorDialogProvider` to your Provider List.
+
+```
+import { ErrorDialogProvider } from "@comet/admin";
+
+const App: React.FunctionComponent = ({ children }) => {
+    return (
+        <OtherProvider>
+            <ErrorDialogProvider>{children}</ErrorDialogProvider>
+        </OtherProvider>
+    );
+};
+```
+
+## Integration into ApolloProvider
+
+There is also the possibility to automatically display an `ErrorDialog` when a graphql error occurs (this can also be disabled for requests, which handle errors on their own).
```

Comment:
```suggestion
There is also the possibility to automatically display an `ErrorDialog` when a GraphQL error occurs (this can also be disabled for requests, which handle errors on their own).
```